### PR TITLE
Laboratory work #3, Sonia Zakharova - 21FPL2

### PR DESCRIPTION
**Pull request by solonarr**

### CODE REVIEW

**Comment by solonarr** [comment]
ломаем и чиним снова и снова

File: config/spellcheck/.wordlist_en.txt [hunk 1]
```diff
@@ -96,7 +96,6 @@ csv
 culture
 cup
 damping
-data
```
Comment (artyomtugaryov) [inline config/spellcheck/.wordlist_en.txt:99]: revert

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
Strange changes. Need to be updated by the mentor

File: config/spellcheck/.wordlist_en.txt (hunk 1 above)
Comment (solonarr) [reply config/spellcheck/.wordlist_en.txt]: done?????????????????????

File: config/stage_1_style_tests/_stage_run_stubgen.sh [hunk 2]
```diff
@@ -18,13 +18,6 @@ for LAB_NAME in $LABS; do
         --target_code_path ./build/stubs/${LAB_NAME}/main.py
 
   check_if_failed
-
```
Comment (artyomtugaryov) [inline config/stage_1_style_tests/_stage_run_stubgen.sh:21]: revert

**Review by artyomtugaryov — COMMENTED** [review]
Please, revert all unrelated changes

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]
looks like something bad in your request

File: lab_3_generate_by_ngrams/main.py [hunk 3]
```diff
@@ -15,6 +16,8 @@ class TextProcessor:
         _end_of_word_token (str): A token denoting word boundary
         _storage (dict): Dictionary in the form of <token: identifier>
     """
+    _end_of_word_token: str
```
Comment (artyomtugaryov) [inline lab_3_generate_by_ngrams/main.py:19]: what? Why?

File: lab_3_generate_by_ngrams/main.py [hunk 4]
```diff
@@ -7,6 +7,7 @@
 from typing import Optional
 
 
+
```
Comment (artyomtugaryov) [inline lab_3_generate_by_ngrams/main.py:10]: discard

File: lab_3_generate_by_ngrams/main.py [hunk 5]
```diff
@@ -159,7 +247,22 @@ def _postprocess_decoded_text(self, decoded_corpus: tuple[str, ...]) -> Optional
...
+        if not isinstance(decoded_corpus, tuple) or len(decoded_corpus) == 0:
+            return None
+
+        result = ''
+
+        for el in decoded_corpus:
+            if el != self._end_of_word_token:
+                result += el
+            else:
+                result += ' '
```
Comment (artyomtugaryov) [inline lab_3_generate_by_ngrams/main.py:259]: fstring

**Review by artyomtugaryov — CHANGES_REQUESTED** [review]

**Pull request closed without merging**